DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
-Upon receiving an ad schedule from Uplynk, you should be able to detect whether or not true[X] is returned in any of the pods. true[X] ads should have `apiFramework` set to `VPAID`.
 
-In the example response, you will see those values at `response.ads.breaks[0].ads[0].apiFramework` and `response.ads.breaks[0].ads[0].creative` respectively.
+### Handling Events from TruexAdRenderer
 
-The Uplynk PrePlay documentation will have more information on determining exactly when the VPAID ad unit should fire -- from what I've ascertained, you'd simply reference `response.ads.placeholderOffsets`, using `adIndex` and `breakIndex` to correlate them to each separate VPAID unit.
+Once `start` has been called on the renderer, it will start to emit events, in the form of calling delegate methods (see [`TruexAdRendererDelegate` Methods -- Main Flow](#truexadrendererdelegate-methods----main-flow) and [`TruexAdRendererDelegate` Methods -- Informative](#truexadrendererdelegate-methods----informative))
 
-Once the player reaches a placeholder, it should pause, instantiate the `TruexAdRenderer` and immediately call `init()` followed by `start()`.
+One of the first events you will receive is [`onAdStarted`](#onadstarted). This notifies the app that the renderer has received an ad for the user and has started to show the unit to the user. The app does not need to do anything in response, however it can use this event to facilitate a timeout. If an `onAdStarted` event has not fired within a certain amount of time after calling `start`, the app can call `stop` on the renderer and proceed to normal video ads.
 
-Alternatively, you can instantiate and `init()` the `TruexAdRenderer` in preparation for an upcoming placeholder. This will give the `TruexAdRenderer` more time to complete its initial ad request, and will help optimize true[X] load time. Once the player reaches a placeholder, it can call `start()` to notify the renderer that it can display the unit to the user.
+At this point, the app must listen for the renderer's [terminal events](#terminal-events), while paying special attention to the [`onAdFreePod`](#onadfreepod) event. A *terminal event* signifies that the renderer is done with its activities and the app may now resume playback of its stream. The `onAdFreePod` event signifies that the user has earned a credit with true[X] and all linear video ads remaining in the current pod should be skipped. If the `onAdFreePod` event did not fire before a terminal event is emitted, the app should resume playback without skipping any ads, so the user receives a normal video ad payload.
 
+#### Terminal Events
 
-### Handling Events from TruexAdRenderer
+The delegate methods representing *terminal events* are:
```

Comment:
This section makes me wonder about the ideal way to handle the responsibilities of documentation between the reference app and here: https://github.com/socialvibe/truex-tvos-google-ad-manager-reference-app#7---respond-to-renderer-finish-events

Perhaps the reference app README should just link back to here if the user wants more info?

Reply:
Yeah, good idea